docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using Teleport CA with Github.
+description:  How to use Teleport SSH short lived certificates with the GitHubs Certificate Authority.
+---
+
+Teleport supports exporting user SSH certificates with configurable key extensions, this allows the Teleport CA to be used in conjunction with GitHubs support for SSH Certificate Authorities to allow users to access their organizations repositories with short lived, signed SSH keys.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Telport with [GitHub SSO configured](../admin/github-sso.mdx).
```

Comment:
```suggestion
- The GitHub SSO authentication connector. For more information, see [GitHub SSO](../admin/github-sso.mdx).
```
Since a Teleport installation is already a prerequisite.